rust/Cargo.toml

```diff
@@ -1,15 +1,13 @@
 [workspace]
-members = [
-    "lib",
-    "android",
-    "zingomobile_utils"
-]
+members = ["lib", "android", "zingomobile_utils"]
 resolver = "2"
 
 [workspace.dependencies]
-zingolib = { git="https://github.com/zingolabs/zingolib", default-features=true, tag = "1.8.1", features=["test-elevation"] }
+zingolib = { git = "https://github.com/zingolabs/zingolib", default-features = true, rev = "cab1ca5b99120bee291d67f37f325715679b29fb", features = [
```

Comment:
I'm thinking about to put a specific commit... but IMO is better to put the `dev` branch... and sometimes to update the Cargo.lock after a `cargo update....` when is necessary.